events/events-team/events-lead.md

```diff
@@ -1,33 +1,84 @@
 # Events Lead Handbook
 
+The Event Lead is responsible for acting as a chair for the event, carrying out
+the strategic vision, and being the glue needed to execute on the event.
+
+- [Overview](#overview)
+- [Skills and Qualifications](#skills-and-qualifications)
+- [Activities](#activities)
+- [Scheduling and Communication Property Access](#scheduling-and-communication-property-access)
+
 ## Overview
 
-The Event Lead is responsible for acting as a chair for the event, carrying out the strategic vision, and being the glue needed to execute on the event.
+The Event Lead(s) is responsible for determining which core roles need to be
+filled at the start of planning for each event. After initial core team meetings,
+each lead will determine how to best staff for that function and fill in other
+roles. Each role will ideally have a shadow for succession planning. Those who
+have been a part of a previous event get first dibs on lead roles to maintain
```

Comment:
nit: First dibs seems a bit... informal for the rolebook and also might not translate well.

Maybe: "part of a previous event is first in line to become the new role lead to maintain continuity"